libs/verification-common/src/blueprint_contracts.rs

```diff
@@ -0,0 +1,101 @@
+use bytes::Bytes;
+
+// ERC5202_PREFIX = b"\xFE\x71\x00"  # default prefix from ERC-5202
+// def blueprint_bytecode(self) -> bytes:
+//     blueprint_bytecode = ERC5202_PREFIX + self.bytecode
+//
+//     # the length of the deployed code in bytes
+//     len_bytes = len(blueprint_bytecode).to_bytes(2, "big")
+//     deploy_bytecode = b"\x61" + len_bytes + b"\x3d\x81\x60\x0a\x3d\x39\xf3"
+//
+//     return deploy_bytecode + blueprint_bytecode
+
+pub fn from_runtime_code(code: Bytes) -> Option<Bytes> {
+    let prefix = [0xfe, 0x71, 0x00];
+    code.starts_with(&prefix)
+        .then(|| code.slice(prefix.len()..))
+}
+
+pub fn from_creation_code(code: Bytes) -> Option<Bytes> {
+    if code.len() < 10 {
+        return None;
+    }
+
+    let deploy_bytecode_prefix = [
+        0x61, code[1], code[2], 0x3d, 0x81, 0x60, 0x0a, 0x3d, 0x39, 0xf3,
```

Comment:
so code[1] + code[2] is length of bytecode, right? could you please add comment describing that